05-UI-ReactJS/ejemplo/src/MoviesList.js

```diff
@@ -0,0 +1,22 @@
+import React, { Component } from 'react'
+import MovieItem from './MovieItem'
+export default class ListMovies extends Component {
+  constructor(props) {
+    super(props)
+  }
+  render() {
+    return (
+      <ul>
+        <span className="movieElement">Title</span>
+        <span className="movieElement">Duration</span>
+        <span className="movieElement">Year</span>
+        {this.props.movies.map((movie, index) => {
+          return (
+            <MovieItem key={index} title={movie.title} duration={movie.duration} year={movie.year} />
```

Comment:
No es buena práctica pasar como key un index. Puede generar errores en el rendering como se dice acá https://reactjs.org/docs/lists-and-keys.html. Podés aprovechar que tus pelis tienen `id`